src/popup/components/TransactionLabel.vue

```diff
@@ -165,6 +165,8 @@ export default defineComponent({
         text = t('transaction.listType.tipReceived');
       } else if (isTip.value) {
         text = t('transaction.listType.tipSent');
+      } else if (innerTx.value.function === TX_FUNCTIONS.verifyAccount) {
```

Comment:
There are potential problems with this approach. A contract made by fraudsters can name its method `verify_account`, and we will have it marked `Verify Account` (the user may be led to believe that the wallet is aware of the contract context/"verified").

Reply:
I removed the verify account additional labels and created separate task for this: https://github.com/superhero-com/superhero-wallet/issues/2897